### PR TITLE
feat: setup wizard

### PR DESCRIPTION
**Pull request by bruno-garcia**
The goal is to have an onboarding flow that will have the user hit Sentry, select an org and project, and will pre-populate the DSN and the Sentry CLI options for symbol upload.

---------------------

Edit @vaind: cleaned this up [_somewhat_](https://github.com/getsentry/sentry-unity/pull/780/files/69b1be294e3a108c45dc10ab08e74f308cdba9bd..0fb1956caa17c6a532e24e285dcecd7072e53c70), resolved TODOs and usability issues. 

Remaining TODO: tests 😱 

Basic workflow:
wizard

----------------

<details>
<summary>First draft does (outdated):</summary>

image

Opens the browser:
image

prepopulates orgs:
image

Then projects:
image

After selecting, the normal window shows up. With DSN, Org and Proj slug and Auth token.

</details>

### CODE REVIEW

**Comment by HazAT** [comment]
If there is only one org, one project
auto fill/select and skip the steps
even better

**Review by bruno-garcia — COMMENTED** [review]
I can't approve my own PR, but great stuff @vaind ! thank you

**Review by bitsandfoxes — APPROVED** [review]

File: src/Sentry.Unity.Editor/ConfigurationWindow/SentryTestWindow.cs [hunk 1]
```diff
@@ -6,7 +6,11 @@ namespace Sentry.Unity.Editor.ConfigurationWindow
 {
     public sealed class SentryTestWindow : SentryWindow, IDisposable
     {
-        protected override string SentryOptionsAssetName { get; } = Path.GetRandomFileName();
+        public SentryTestWindow()
```
[the remark this answers is not in the document]
Comment (vaind) [reply src/Sentry.Unity.Editor/ConfigurationWindow/SentryTestWindow.cs:9]: moved

File: Directory.Build.targets [hunk 2]
```diff
@@ -552,5 +552,9 @@ void PrintFailedTests(XElement element)
     <Message Importance="High" Text="Replacing $(SentryArtifactsDestination)%(SDK.Identity)" />
     <RemoveDir Directories="$(SentryArtifactsDestination)%(SDK.Identity)" />
     <Exec Command="gh run download $(LastSuccessfulRunId) -n &quot;%(SDK.Identity)-sdk&quot; -D &quot;$(SentryArtifactsDestination)%(SDK.Identity)&quot;" />
+
+    <!-- Download overwrites some files that then show up as chagned in the IDE, even though there are no changes, e.g. only whitespace -->
```
[the remark this answers is not in the document]
Comment (vaind) [reply Directory.Build.targets:554]: thx

File: src/Sentry.Unity.Editor/ConfigurationWindow/Wizard.cs [hunk 3]
```diff
@@ -0,0 +1,359 @@
...
+            if (Instance is null)
+            {
+                Instance = ScriptableObject.CreateInstance<Wizard>();
+                Instance._logger = logger;
+
+                SentryWindow.SetTitle(Instance, description: "Setup wizard");
+
+                Instance.ShowUtility();
+                Instance.minSize = new Vector2(600, 200);
+                Instance.StartLoder();
```
[the remark this answers is not in the document]
Comment (vaind) [reply src/Sentry.Unity.Editor/ConfigurationWindow/Wizard.cs:38]: thx

File: src/Sentry.Unity.Editor/ConfigurationWindow/Wizard.cs [hunk 4]
```diff
@@ -0,0 +1,359 @@
...
+            }
+        }
+
+        private void StartLoder()
```
[the remark this answers is not in the document]
Comment (vaind) [reply src/Sentry.Unity.Editor/ConfigurationWindow/Wizard.cs:42]: thx

**Pull request merged**